### ISSUE TITLE
Chrome forces .dev domains to use SSL. Use .localhost instead

### ISSUE THREAD

**Issue by generalredneck** [geerlingguy/drupal-vm#1642]
### Issue Type

 - Documentation Report

### Summary

As written in https://ma.ttias.be/chrome-force-dev-domains-https-via-preloaded-hsts/ Chrome has forced .dev and .foo domains to load SSL as Google has bought .dev. It has been suggested to change to .localhost

I write this as warning / suggestion to change default.config.yml to use .localhost instead of .dev in future versions as there is likely to be real .dev sites showing up in the wild as well as a the confusing part that our default http only drupalvm is now trying to load via ssl because of chrome.

**Comment by generalredneck** [comment]
FYI, this is chrome >=63

**Comment by lopezs** [comment]
Thank you! This has been driving me nuts all day.

Turns out the default was changed from .dev to .test a few months ago,  #1427 

Confirming that changing my projects to .test fixes the problem.

**Comment by generalredneck** [comment]
Good Call. Thanks for double checking my issue. Looks like the one I was testing was on 4.6.0. Marking as closed.
